### DISCUSSION TITLE
a question about Tle2Orbit

### DISCUSSION THREAD

**Discussion by zdwangjian5** [groops-devs/groops#286] [Q&A] [answered]
I want to use Tle2Orbit program to predict the orbit of GRACE-FO, the tle is from the celestrak website. But it always return a error like this:
![image](https://github.com/user-attachments/assets/cfbf6e83-44bd-4bd5-bb22-581f8482a559)

![image](https://github.com/user-attachments/assets/2568b7f3-f043-4a70-b3e5-c0115292e11b)
 Looking for your help,thank you!
0 GRACE-FO 1
1 43476U 18047A   24271.15884463  .00006933  00000-0  23169-3 0  9994
2 43476  88.9699 284.9868 0014629  45.9092 314.3360 15.30757204353466

**Comment by tmayerguerr** [comment, marked as answer]
Hi,
As stated in the error message:
- in earthOrientationParameter
- in interpolate
- cannot interpolate at 2024-09-20_00-00-00

=> You should update update your EOP file (timeSeries_EOP_rapid_IAU2000_desai.dat) to get the current EOPs.
Torsten

**Reply by zdwangjian5** [reply]
Thank you!
